assets/styles/index.js

```diff
@@ -0,0 +1,13 @@
+import boxItemStyle from './boxItem';
+import twoColumnViewStyle from './twoColumnView';
+import appStyle from './appStyle';
+import detailScreen from './detailScreen';
+import * as loaderStyle from './loaderStyle';
+
+export {
```

Comment:
Actually u don't need to do this again.
Since in each of your individual style.js u have exported the function, this is a repetition.

Reply:
I know, I could just use `../styles/...` import but I wanted the code to be available like this (for example):
```javascript
import { loaderStyle } from '../assets/styles'
```
and not like this (which you can still use 😄):
```javascript
import loaderStyle from '../assets/styles/loaderStyle'
```
It's up to you which one you want to use.

Reply:
for  `import { loaderStyle } from '../assets/styles'` to be used, you do `export loaderStyle` instead of `export default loaderStyle`

Reply:
Oh, I know what you mean (I'll fix it now)! For the duplication - styles were already done, the point was just to move them to separate files - that's why it all looks so similar.